oneClickInstaller.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+#check requisites
+sh ./checkPreRequisites.sh
```

Comment:
looks like you forgot to include `checkPreRequisites.sh` file.

Reply:
I forgot to add it as a file  - I added it and committed it, can you see it now or should I do a new pull request?

Reply:
Yes I can see it now. 👍